src/com/wolox/parser/ConfigParser.groovy

```diff
@@ -59,27 +59,35 @@ class ConfigParser {
 
         def tasks = [:]
         yamlTasks.each { k, v ->
-            def osEntries = [:]
-            if (v.os) {
-                osEntries = v.os.collectEntries()
+
+            def osEntries = []
+            if ( v.os) {
+                osEntries = v.os
             } else {
-                osEntries.put DEFAULT_LX_NODE, DEFAULT_OS
```

Comment:
why do you change from .put? I think osEntries.put ("DEFAULT_LX_NODE", "DEFAULT_OS"); would have also worked